src/start.js

```diff
@@ -15,9 +16,18 @@ function startServer({ port = process.env.PORT } = {}) {
 
   app.use(errorMiddleware)
 
+  async function initilizeLatestPatch() {
+    const latest = await riotApiClient.initialize()
+    return latest
+
+  }
+  
+
+  
   return new Promise((resolve) => {
-    const server = app.listen(port, () => {
+    const server =  app.listen(port, () => {
       logger.info(`Listening on port ${server.address().port}`)
+      initilizeLatestPatch()
```

Comment:
This should not go here. The startServer function should do one thing, which it did, start the server. The call to riotApiClient.initialize() should occur right after startServer() is called from the index.js function.